pre_install_report/library/utils/hello-ingress-k8s-v118.yaml

```diff
@@ -0,0 +1,27 @@
+####################################################################
+####   hello-ingress.yml                                        ####
+####################################################################
+####################################################################
+#### Author: SAS Institute Inc.                                                                                       ####
+####                                                                                                                                           ####
+####################################################################
+####################################################################
+#
+# Copyright (c) 2019-2021, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
```

Comment:
Copyright on a new file shouldn't include 2019.  Just 2021.

Reply:
Completed update of Copyright